gradle.properties

```diff
@@ -5,7 +5,7 @@ org.gradle.jvmargs=-XX:MaxPermSize=512m -XX:+CMSClassUnloadingEnabled -XX:+CMSPe
 bundleInstallRoot = /apps/grabbit/install
 
 group = com.twcable.grabbit
-version = 4.0.4
+version = 4.0.5
```

Comment:
Version changes should happen as their own commit per [RELEASING.adoc](../../blob/master/docs/RELEASING.adoc)